mobile/src/board/components/BoardView.tsx

```diff
@@ -57,13 +72,57 @@ const BoardView = ({
         )}
       </div>
 
-      <TileView
-        boardTile={remainingTile}
-        canPlay={canPlay}
-        gameState={gameState}
-        onRotateRemainingTile={onRotateRemainingTile}
-        onInsertTile={onInsertTile}
-      />
+      <Box width={"100%"}>
+        <Grid container spacing={2}>
+          <Grid
+            item
+            xs={4}
+            display={"flex"}
+            alignItems={"center"}
+            justifyContent={"center"}
+          >
+            <TileView
+              boardTile={remainingTile}
+              canPlay={canPlay}
+              gameState={gameState}
+              onRotateRemainingTile={onRotateRemainingTile}
+              onInsertTile={onInsertTile}
+              onMovePlayer={onMovePlayer}
+              playerTarget={user?.currentTarget}
+            />
+          </Grid>
+          <Grid item xs={8}>
+            {user && (
+              <Stack>
```

Comment:
The Stack is probably useless here. I didn't see you had a more complicated layout and the Grid alone is enough, right?

Reply:
The stack is effectively useless, I removed it